ipykernel/iostream.py

```diff
@@ -445,9 +445,7 @@ def _schedule_flush(self):
 
         # add_timeout has to be handed to the io thread via event pipe
         def _schedule_in_thread():
-            # FIXME: original code was:
-            # self._io_loop.call_later(self.flush_interval, self._flush)
-            self._io_loop.call_soon_threadsafe(self._flush)
+            self._io_loop.call_soon_threadsafe(self._io_loop.call_later, self.flush_interval, self._flush)
```

Comment:
Not sure if `call_soon_threadsafe` is needed. If not, we could keep the original code.